scripts/build-minimal-linux.sh

```diff
@@ -0,0 +1,8 @@
+docker run --rm \
+ -v /var/run/docker.sock:/var/run/docker.sock \
+ -v $(which docker):$(which docker) \
+ -v /Users:/Users \
```

Comment:
Hi @doublerr!

Thanks for contributing this PR.  I suggest changing this from `/Users` to `$(pwd)`.  Once that change is made, this build script also works on Linux.